psi4/driver/procrouting/dft_funcs/superfuncs.py

```diff
@@ -194,6 +200,14 @@ def build_superfunctional(name, restricted):
         else:
             for x_func in sup[0].x_functionals():
                 x_func.set_omega(omega)
+
+    # add VV10 correlation to DFT aka DFT-NL (Hujo/Grimme) 
+    if core.has_option_changed("SCF", "DFT_VV10_B"):
+        vv10_b = core.get_option("SCF", "DFT_VV10_B")
+        sup[0].set_vv10_b(vv10_b)
+        sup[0].set_vv10_c(0.0093) # unchanged in DFT-NL scheme
```

Comment:
`dashparams` should be already set up to allow on-the-fly parameter changes, e.g., https://github.com/psi4/psi4/blob/master/tests/dftd3/energy/input.dat#L93-L103. Will have to be hooked up to your code rather than the dftd3-runner.